kafl_gui.py

```diff
@@ -247,123 +250,184 @@ def draw(self, cur_rows=None):
-            (16, "Timeouts", pfloat(d.relative_timeouts()) + "%"),
-            (15, "RAM Use", "%3d%%" % d.ram_used())])
+        self.gui.print_info_line(
+            [
+                (16, "Runtime", ptime(d.runtime())),
+                (16, "#Execs", pnum(d.total_execs())),
+                (16, "Stability", "%3d%%" % d.stability()),
+                (15, "Workers", "%d/%d" % (d.num_workers(), d.cpu_cores())),
+            ]
+        )
+        self.gui.print_info_line(
+            [
+                (16, "", ""),
+                (16, "CurExec/s", pnum(d.execs_p_sec_cur())),
+                (16, "Funkiness", pfloat(d.relative_funky()) + "%"),
+                # (16, "Reload/s", pnum(d.reload_p_sec()))])
+                (15, "CPU Use", "%3d%%" % d.cpu_used()),
+            ]
+        )
+        self.gui.print_info_line(
+            [
+                (16, "Est. Done", "%3d%%" % d.est_done()),
+                (16, "AvgExec/s", pnum(d.execs_p_sec_avg())),
+                (16, "Timeouts", pfloat(d.relative_timeouts()) + "%"),
+                (15, "RAM Use", "%3d%%" % d.ram_used()),
+            ]
+        )
         self.gui.print_end_line()
         self.gui.print_header_line("Progress")
-        self.gui.print_info_line([
-            (16, "Paths", ""),
-            (16, "Bitmap", ""),
-            (35, "Findings", "")])
-        self.gui.print_info_line([
-            (16, " Total", pnum(d.paths_total())),
-            (16, "", ""),
-            (35, " Crash", "%6s (N/A) %10s" % (pnum((d.num_found("crash"))),
-                                                 ptime(d.time_since("crash"))))])
-        self.gui.print_info_line([
-            (16, " Seeds", pnum(d.yield_imported())),
-            (16, " Edges", pnum(d.bitmap_used())),
-            (35, " AddSan", "%6s (N/A) %10s" % (pnum((d.num_found("kasan"))),
-                                                 ptime(d.time_since("kasan"))))])
-        self.gui.print_info_line([
-            (16, " Favs", pnum(d.fav_total())),
-            (16, " Blocks", pnum(d.bb_covered())),
-            (35, " Timeout", "%6s (N/A) %10s" % (pnum((d.num_found("timeout"))),
-                                                 ptime(d.time_since("timeout"))))])
-        self.gui.print_info_line([
-            (16, " Norm", pnum(d.normal_total())),
-            (16, " p(col)", pfloat(d.p_coll()) + "%"),
-            (35, " Regular", "%6s (N/A) %10s" % (pnum((d.num_found("regular"))),
-                                                 ptime(d.time_since("regular"))))])
+        self.gui.print_info_line([(16, "Paths", ""), (16, "Bitmap", ""), (35, "Findings", "")])
+        self.gui.print_info_line(
+            [
+                (16, " Total", pnum(d.paths_total())),
+                (16, "", ""),
+                (
+                    35,
+                    " Crash",
+                    "%6s (N/A) %10s" % (pnum((d.num_found("crash"))), ptime(d.time_since("crash"))),
```

Comment:
no....I know this code is crap, but alignment is all that makes it readable.
Need to disable for this whole function/file or refactor..